form/root_storage/root_rfield_write_container.cpp

```
@@ -0,0 +1,130 @@
+  {
+    this->Storage_Associative_Write_Container::setParent(parent);
+    auto parentDerived = dynamic_pointer_cast<ROOT_RNTuple_Write_ContainerImp>(parent);
+    if (!parentDerived) {
+      throw std::runtime_error("ROOT_RField_Write_ContainerImp::setParent parent is not a "
+                               "ROOT_RNTuple_Write_ContainerImp!  Something "
+                               "may be wrong with how Storage works.");
+    }
+    m_rntuple_parent = parentDerived;
+  }
+
+  void ROOT_RField_Write_ContainerImp::fill(void const* data)
```

Cppcheck (2.20.0)

[style] The function 'fill' is never used.

(unusedFunction)